pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/WorkPredicates.scala

```diff
@@ -11,27 +11,40 @@ import uk.ac.wellcome.models.work.internal.{
 object WorkPredicates {
   type WorkPredicate = TransformedBaseWork => Boolean
 
-  val sierraWork: WorkPredicate = identifierTypeId("sierra-system-number")
-  val metsWork: WorkPredicate = identifierTypeId("mets")
-  val miroWork: WorkPredicate = identifierTypeId("miro-image-number")
-
+  private val sierraIdentified: WorkPredicate = identifierTypeId(
```

Comment:
Why private?

Reply:
My logic was they're not intended for use in the Rules.
But I suppose could be if need be.